src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Sample Angular App</title>
+    <title>Foundation X</title>
```

Comment:
Get these text values from the i18n language json file instead with translation service.
We would also need to maintain a document which would include things a  user need to replace like name of the app, hero.png etc..

Reply:
@xixas 

1. Yes, internationalization was planned to be covered later. I will add a new commit to this PR and implement that along with it.
2. Changes may vary broadly depending on the project's requirements. I don't think there is much requirement to add such doc that says what to replace and all. Would like to discuss more on this.

Reply:
I have added localization feature to the app in my latest commit. The localization has following features

1. For the first time visitor detects browser language and translate the app to that language if available.
2. User get an option to choose the languages in the footer.
3. The app gets translated to chosen language and the preference gets saved until the user changes it.
![translation-en](https://github.com/OsmosysSoftware/angular-foundation/assets/108815380/ac0acd26-11b1-4eb9-a384-67f29a795e9d)


![translation-fr](https://github.com/OsmosysSoftware/angular-foundation/assets/108815380/d0afe228-f54a-4122-ad7f-d8fa52f25ee1)